src/ceph-disk

```diff
@@ -2461,7 +2464,7 @@ def get_partition_type(part):
 
 
 def get_partition_uuid(dev):
-    if 'loop' in dev or 'cciss' in dev:
+    if 'loop' in dev or 'cciss' in dev or 'nvme' in dev:
         (base, partnum) = re.match('(.*\d+)p(\d+)', dev).group(1, 2)
     else:
         (base, partnum) = re.match('(\D+)(\d+)', dev).group(1, 2)
```

Comment:
Could you make these lines into a **split_dev_base_partnum** function 
<pre>
def split_dev_base_partnum(dev):
    if 'loop' in dev or 'cciss' in dev or 'nvme' in dev:
         return re.match('(.*\d+)p(\d+)', dev).group(1, 2)
     else:
         return re.match('(\D+)(\d+)', dev).group(1, 2)
</pre>
that can later be called with
<pre>
(base, partnum) = split_dev_base_partnum(dev)
</pre>